app/views/account_reset/request/show.html.erb

```diff
@@ -1,27 +1,62 @@
-<% self.title = t('account_reset.request.title') %>
+<% if IdentityConfig.store.updated_account_reset_content %>
+  <% self.title = t('account_reset.request.delete_account') %>
 
-<%= render PageHeadingComponent.new.with_content(t('account_reset.request.title')) %>
+  <%= render PageHeadingComponent.new.with_content(t('account_reset.request.delete_account')) %>
+  <p><%= t(
+           'account_reset.request.delete_request_details_html',
+           account_delete_link: new_tab_link_to(
+             t('account_reset.request.account_delete_link'),
+             MarketingSite.help_center_article_url(
+               category: 'manage-your-account',
+               article: 'delete-your-account',
+             ),
+           ),
+         ) %></p>
+  
+  <p><%= t('account_reset.request.choose_to_continue') %></p>
 
-<% t('account_reset.request.info').each do |info_p| %>
-  <p><%= info_p %></p>
-<% end %>
+  <p><%= t('account_reset.request.delete_email1_html') %></p>
+  <p><%= t('account_reset.request.delete_email2_html', interval: @account_reset_deletion_period_interval) %></p>
 
-<h2><%= t('account_reset.request.delete_account') %></h2>
+  <p><strong><%= t('account_reset.request.continue_deletion') %></strong></p>
+  <%= button_to(
+        account_reset_request_path,
+        form_class: 'margin-top-4 margin-bottom-2',
+        class: 'usa-button usa-button--big usa-button--wide',
+        method: :post,
+      ) { t('account_reset.request.yes_continue') } %>
+  <%= button_to(
+        root_url,
+        method: :get,
+        class: 'usa-button usa-button--unstyled',
+      ) { t('account_reset.request.no_cancel2') } %>
+  
+<% else %>
+  <% self.title = t('account_reset.request.title') %>
 
-<% t('account_reset.request.delete_account_info', interval: @account_reset_deletion_period_interval).each do |info_p| %>
-  <p><%= info_p %></p>
-<% end %>
+  <%= render PageHeadingComponent.new.with_content(t('account_reset.request.title')) %>
```

Comment:
Why are we rendering two `PageHeadingComponent` partials?

Reply:
yes its an if else statement. but maybe we can keep them separate.